apps/src/templates/SchoolZipSearch.jsx

```diff
@@ -35,12 +39,29 @@ export default function SchoolZipSearch({fieldNames, zip}) {
       });
   }, [zip]);
 
+  const sendAnalyticsEvent = (eventName, data) => {
+    analyticsReporter.sendEvent(eventName, data, PLATFORMS.BOTH);
+  };
+
   const onSchoolChange = e => {
     const schoolId = e.target.value;
-    setSelectedSchoolNcesId(schoolId);
-    if (schoolId === 'clickToAdd') {
+    let ncesId;
+    if (schoolId === NO_SCHOOL_SETTING) {
+      ncesId = '';
+      sendAnalyticsEvent(EVENTS.DO_NOT_TEACH_AT_SCHOOL_CLICKED, {});
+    } else if (schoolId === SELECT_A_SCHOOL) {
+      ncesId = '';
+      // sendAnalyticsEvent(EVENTS.SCHOOL_LIST_OPENED, {});
```

Comment:
should this be commented out?

Reply:
Yes- there's a nod to it in the PR description.. I'm waiting to hear back from Dani about how this should go so I have a placeholder here for now (to get the other events sending).

Reply:
🤦‍♀️ sorry, reading too fast trying to clear out reviews before the end of my day. Makes sense.

Reply:
All good!!